recipe/meta.yaml

```diff
@@ -6,27 +6,28 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: a0258537e65e7e14ec33a86802f865a667f949bb6c73646d55e42f7c45a052ae
 
 build:
-  number: 0
+  number: 1
   script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
   noarch: python
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python {{ python_min }}
   run:
     - gast >=0.5.0,<0.6.0
```

Comment:
To be checked but it seems to me that Beniget 0.4.2 is also compatible with gast 0.6.0.